tools/USAGE.md

```diff
@@ -0,0 +1,63 @@
+# Raml Toolkit Tools
+
+A tool to lint & verify all commerce-sdk endpoints in an isolated container. 
+
+- [Pre-Requisites](#usage)
+- [Usage](#usage)
+
+## Pre-Requisites
+
+This tool uses docker to lint and verify all commerce-sdk endpoints. 
+The tool requires [raml-toolkit](https://github.com/SalesforceCommerceCloud/raml-toolkit) and 
+[commerce-sdk](https://github.com/SalesforceCommerceCloud/commerce-sdk) repositories cloned to your local computer. 
+
+* Install [Docker Engine](https://docs.docker.com/engine/install/) 
+* Git clone [raml-toolkit](https://github.com/SalesforceCommerceCloud/raml-toolkit) repository
+* Git clone [commerce-sdk](https://github.com/SalesforceCommerceCloud/commerce-sdk) repository
+
+## Usage
+
+The docker container requires `raml-toolkit` directory mounted as `/linter` volume and 
+`commerce-sdk/apis` directory mounted as `/apis` volume. You can do so as follows
+
+```shell script
+$ docker run -t -d --name linter -v /<<raml-toolkit absolute path>>:/linter -v /<<commerce-sdk/apis absolute path>>:/apis node
```

Comment:
Best practice is to use a tag for the image to make sure this stays stable. I'd recommend 12-alpine
Also, you should be able to just add sh /linter/tools/lint.sh onto the end of this and skip the second command

Reply:
do you mean a docker-compose file? It could just be another shell script

Reply:
not docker-compose, just the docker file that has instructions to run the linter inside docker instead of mounting

Reply:
I think you mean just having another shell script. There's not a very good reason to build a custom docker image.

Reply:
Docker file is a little heavy for this one.. In the latest change, i even deleted the shell script

Pls take a look

Reply:
Yup, looks very good. One other thing to consider is adding a note about storing the command as a bash alias.
```
alias linter-test="docker run -t --name linter -v /<<raml-toolkit absolute path>>:/linter -v /<<commerce-sdk/apis absolute path>>:/apis node:12-alpine /linter/tools/lint.js"
linter-test /linter/commerce-apps-raml-toolkit-0.3.1.tgz
```